### PR TITLE
Tutorials, Tech Assistant, Suggestion Servlet

### PR DESCRIPTION
**Pull request by nihabhaskar**
Creates files Suggestion.java, SuggestionServlet.java, techassistant.java, tutorials.html, zoom.html. 

Suggestion.java - Entity type I am adding into datastore for suggestions made on tutorials page.
SuggestionServlet.java - Stores suggestions made on tutorials page.
techassistant.java - Page for seniors to navigate different resources for tech help. Basic html page for navigating either to tutorials or to forum.
tutorials.html - Lists all available tutorials. Users can ask suggestions. Will contain Youtube video on how to navigate the page.
zoom.html - first sample page of content on a tutorial page.

### CODE REVIEW

File: src/main/java/com/google/sps/data/Suggestion.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  private final String platform;
+  private final String addition;
+  private final long timestamp;
+
+  public Suggestion( String platform, String addition, long timestamp) {
+    this.platform = platform;
+    this.addition = addition;
+    this.timestamp = timestamp;
+  }
+}
```
Comment (geoffreyzgoog) [inline src/main/java/com/google/sps/data/Suggestion.java:15]: Style nit: Always add a line break at the end. (Add to code linter if possible.)

File: src/main/java/com/google/sps/servlets/SuggestionServlet.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    suggestionEntity.setProperty("timestamp", timestamp);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(suggestionEntity);
+
+    response.sendRedirect("/index.html");
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {}
```
Comment (geoffreyzgoog) [inline src/main/java/com/google/sps/servlets/SuggestionServlet.java:34]: If get is not supported, then throw an exception.

File: src/main/java/com/google/sps/data/Suggestion.java (hunk 1 above)
Comment (nihabhaskar) [reply src/main/java/com/google/sps/data/Suggestion.java]: Added line breaks to end of files.

File: src/main/java/com/google/sps/servlets/SuggestionServlet.java (hunk 2 above)
Comment (nihabhaskar) [reply src/main/java/com/google/sps/servlets/SuggestionServlet.java]: Removed the doGet method all together.

**Comment by nihabhaskar** [comment]
I am not sure what some of the html linting errors are referring to; for example "attribute values cannot be empty". I know some of the errors refer to elements with "id" not existing I am not working on those pages (forum.html & tech-survey.html) but I assume it will be fixed in the newer version of these pages @rmr167 @praneetmek .

**Comment by nihabhaskar** [comment]
Removed files that will be fixed with later PR's, still not sure what "attribute cannot be empty" means.

**Comment by geoffreyzgoog** [comment]
> I am not sure what some of the html linting errors are referring to; for example "attribute values cannot be empty". I know some of the errors refer to elements with "id" not existing I am not working on those pages (forum.html & tech-survey.html) but I assume it will be fixed in the newer version of these pages @rmr167 @praneetmek .

I think that's due to the spaces around attribute assgiment like "class = " and "id = ".

**Comment by nihabhaskar** [comment]
> > I am not sure what some of the html linting errors are referring to; for example "attribute values cannot be empty". I know some of the errors refer to elements with "id" not existing I am not working on those pages (forum.html & tech-survey.html) but I assume it will be fixed in the newer version of these pages @rmr167 @praneetmek .
> 
> I think that's due to the spaces around attribute assgiment like "class = " and "id = ".

Ah ok thank you.

File: src/main/webapp/techassistant.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <div id="buttons">
+      <div id = "left-tutorials-button">
+        <button class="tech-button" onclick="document.location='/tutorials.html'">Tutorials</button>
+      </div>
+
+      <div id="right-forum-button">
+        <button class="tech-button" onclick="document.location='/forum.html'">Forum</button>
+      </div>
+
+      <div class="clearfix"></div>
```
Comment (rmr167) [inline src/main/webapp/techassistant.html:28]: What is the clearfix for?

File: src/main/webapp/techassistant.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Seniors Connect</title>
+  <link rel="stylesheet" href="styles/techstyle.css">
+  <script src="script.js"></script>
```
Comment (rmr167) [inline src/main/webapp/techassistant.html:7]: Do we need to include script.js?

File: src/main/webapp/zoom.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  <div id="nav-placeholder"> </div>
+
+  <div class="content">
+    <div class="header">
+      <h1>Zoom</h1>
+      <p>Description of Zoom</p>
+      <a href="https://zoom.us/"></a>
+    </div>
+
+    <div id = "helpful-links">
```
Comment (rmr167) [inline src/main/webapp/zoom.html]: Styling: Spaces around the equal sign.  Not quite sure where the styling reference is from but we discussed this during our stand up meeting.

File: src/main/webapp/zoom.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      <li><a href="https://support.zoom.us/hc/en-us/articles/206618765-Zoom-video-tutorials">Zoom Video Tutorials</a></li>
+      <li><a href="https://support.zoom.us/hc/en-us/articles/201362193-Joining-a-meeting">Joining a Meeting</a></li>
+      <li><a href="https://support.zoom.us/hc/en-us/articles/201362613-How-Do-I-Host-A-Video-Meeting-">Hosting a Meeting</a></li>
+    </div>
+
+    <div id = "helpful-videos">
```
Comment (rmr167) [inline src/main/webapp/zoom.html]: Styling: Same as above. Spaces around the equal sign.

File: src/main/webapp/techassistant.html (hunk 3 above)
Comment (nihabhaskar) [reply src/main/webapp/techassistant.html:28]: Oops this was left over from some earlier styling I was trying with this page. Removed it. 
--

Comment (nihabhaskar) [reply src/main/webapp/techassistant.html:28]: Oops this was leftover from some styling I was trying earlier. Removed it now.

File: src/main/webapp/techassistant.html (hunk 4 above)
Comment (nihabhaskar) [reply src/main/webapp/techassistant.html:7]: Was not sure if this page would eventually need a script file so I just kept it, but I can remove it.

File: src/main/webapp/zoom.html (hunk 5 above)
Comment (nihabhaskar) [reply src/main/webapp/zoom.html]: Fixed.

File: src/main/webapp/zoom.html (hunk 6 above)
Comment (nihabhaskar) [reply src/main/webapp/zoom.html]: Fixed.

File: target/maven-archiver/pom.properties [hunk 7]
```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
```
Comment (praneetmek) [inline target/maven-archiver/pom.properties:1]: idt ur .gitignore is working properly. There shouldn't be any changes to target from ur local branch.

Comment (nihabhaskar) [reply target/maven-archiver/pom.properties:1]: Hmm ok I will try recreating it.

Comment (nihabhaskar) [reply target/maven-archiver/pom.properties]: Should be working now.

File: src/main/webapp/techassistant.html (hunk 4 above)
Comment (rmr167) [reply src/main/webapp/techassistant.html]: For now it probably isnt necessary to include.  We could add it back later.

Comment (nihabhaskar) [reply src/main/webapp/techassistant.html]: removed.

**Pull request merged**